scripts/test-check-packages.sh

```diff
@@ -45,7 +45,7 @@ cd -
 elastic-package stack up -d -v
 
 # Boot up the kind cluster
-kind create cluster
+wget -qO-  https://raw.githubusercontent.com/elastic/elastic-package/master/scripts/kind-config.yaml | kind create cluster --config -
```

Comment:
Umm, this makes all builds depend on the configuration in master, what will be problematic if we want to introduce and test a new version of the config in a PR and/or run builds with previous code. Is there any reason to don't take the config directly from `scripts/kind-config.yaml`? Something like this:
```suggestion
kind create cluster --config $CURRENT_DIR/kind-config.yaml
```

In the docs it may be ok to leave the reference to master as we expect this to be a working config, and the user might not have a working copy of this repo.

Reply:
Well, it is actually being already problematic :slightly_smiling_face: https://github.com/elastic/elastic-package/pull/322#issuecomment-819373897

Reply:
In this case we'll have to keep it in both places (elastic/integrations and elastic/elastic-package). The question is which option is less disturbing :)

Reply:
Well, if `elastic/integrations` also needs this small config, I'd rather have it duplicated there than adding a dependency to a delopment branch :slightly_smiling_face: 

Also the needings in both repos may be different, so they may need in any case a different config at some point.

Reply:
I don't have any strong preferences here. The file resource in both places is also acceptable.